### PR TITLE
model equipment created

### PR DESCRIPTION
**Pull request by TonyBM**

### CODE REVIEW

File: app/models/equipment.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Equipment
+  include Mongoid::Document
+  include Mongoid::Enum
+
+  field :description, type: String
+  field :issue, type: String
+  field :observation, type: String
+  enum :status, %i[available,unavailable]
```
Comment (Arcanm) [inline app/models/equipment.rb]: sin coma

Comment (TonyBM) [reply app/models/equipment.rb]: corregido

File: app/models/equipment.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Equipment
+  include Mongoid::Document
+  include Mongoid::Enum
+
+  belongs_to :model
```
Comment (miguelhuchim05) [inline app/models/equipment.rb]: mover la asociación debajo de fields.

orden: Includes, Callbacks, Fields, Associations, Validations, Scopes, Methods

Comment (TonyBM) [reply app/models/equipment.rb]: Corregido

**Pull request merged**